native/src/widget/pane_grid.rs

```diff
@@ -756,7 +757,12 @@ pub fn draw<Renderer, T>(
         cursor_position
     };
 
-    for ((id, pane), layout) in elements.zip(layout.children()) {
+    // Render picked pane last
+    let mut elements = elements.zip(layout.children()).collect::<Vec<_>>();
+    elements
+        .sort_by_key(|((id, _), _)| picked_pane.map(|(id, _)| id) == Some(*id));
```

Comment:
Allocating and sorting seems a bit overkill. Maybe we could loop for the non-picked panes first, and then execute a single iteration for the picked one?

Reply:
Agreed, let me fix

Reply:
Resolved in 470723c. No reason to iterate twice, I just store the needed info in an Option and render it after the iterations

Reply:
Nice :tada: